site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1068,6 +1071,7 @@ public function exportTeams($gradeable_id) {
                         $user->getDisplayedLastName(),
                         $user->getId(),
                         $team->getId(),
+                        $team->getTeamName(),
```

Comment:
how is `$team->getTeamName()` get encoded within the CSV when null? `''`?

Reply:
An example would be: `Alyssa P,Hacker,aphacker,00006_aphacker,,1,4`. There is just simply two commas next to each other which creates an empty record for team name.

Reply:
and then on import, is that read in as empty string or null?

Reply:
I just tested it and it seems it is read and imported as an empty string. I will need to change it so it converts to null if it is an empty string.